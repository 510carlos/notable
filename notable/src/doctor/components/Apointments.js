import React from 'react';
import { Table } from 'antd';


const columns = [
  {
    title: 'Name',
    dataIndex: 'name',
    key: 'name',
  },
  {
    title: 'Time',
    dataIndex: 'time',
    key: 'time',
  },
  {
    title: 'Kind',
    dataIndex: 'kind',
    key: 'kind',
  }
];

const Apointments = ({apt}) =>
  <Table columns={columns} dataSource={apt} />;

export default Apointments;