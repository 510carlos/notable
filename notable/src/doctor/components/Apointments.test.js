import { shallow, mount } from 'enzyme';
import enzymeToJson from 'enzyme-to-json';
import * as React from 'react';

import Apointments from './Apointments';

const props = [{"key":"1","name":"3 Sterling Archer","time":"8:00AM","kind":"New Patient"},{"key":"2","name":"Cyril Figis","time":"8:30AM","kind":"Follow Up"},{"key":"3","name":"Ray Gillete","time":"9:00AM","kind":"Follow Up"},{"key":"4","name":"Lana Kane","time":"9:30AM","kind":"New Patient"},{"key":"5","name":"Pam Poovey","time":"10:00AM","kind":"New Patient"}];

const dom = <Apointments {...props} />

describe('Unit tests for Apointments component', () => {
    it('Renders', () => {
        shallow(dom);
    })

    it('Renders with snapshot', () => {
        const wrapper = shallow(dom);
        expect(enzymeToJson(wrapper)).toMatchSnapshot();
    })
})