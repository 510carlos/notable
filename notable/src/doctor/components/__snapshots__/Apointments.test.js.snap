// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit tests for Apointments component Renders with snapshot 1`] = `
<Table
  bordered={false}
  childrenColumnName="children"
  className=""
  columns={
    Array [
      Object {
        "dataIndex": "name",
        "key": "name",
        "title": "Name",
      },
      Object {
        "dataIndex": "time",
        "key": "time",
        "title": "Time",
      },
      Object {
        "dataIndex": "kind",
        "key": "kind",
        "title": "Kind",
      },
    ]
  }
  dataSource={Array []}
  indentSize={20}
  loading={false}
  locale={Object {}}
  rowKey="key"
  showHeader={true}
  size="default"
  sortDirections={
    Array [
      "ascend",
      "descend",
    ]
  }
  useFixedHeader={false}
/>
`;
